src/manifest-converter.ts

```diff
@@ -92,55 +93,155 @@ export function writeJson(json: any, ...pathPieces: string[]) {
+  };
+
+  if (!packageJson.main) {
+    let main;
+    if (typeof bowerJson.main === 'string') {
+      main = bowerJson.main;
+    } else if (Array.isArray(bowerJson.main)) {
+      if (bowerJson.main.length === 1) {
+        main = bowerJson.main[0];
+      } else {
+        // We could potentially be smarter here. Bower configs have loose
+        // semantics around main, and allow one file per filetype. There might
+        // be an HTML file for the main element, and some extra things like a
+        // CSS file. Maybe in that case we should find just the HTML file.
+        //
+        // There could also be multiple HTML files in main, e.g. a repo like
+        // paper-behaviors which contains 3 separate Polymer behaviors. We
+        // currently let that be, so importing the module directly will fail.
+        // We could also generate an index that re-exports all of the symbols
+        // from all of the mains.
+        console.warn(
+            `${bowerJson.name}: Found multiple mains in bower.json, ` +
+            `but package.json must have only one.`);
+      }
+    }
+    if (main && main.endsWith('.html')) {
+      // Assume that the bower main is already a correct relative path to an
+      // HTML file, and that the module equivalent will be in the same directory
+      // but with a JS extension.
+      packageJson.main = replaceHtmlExtensionIfFound(main);
+    } else {
+      console.warn(
+          `${bowerJson.name}: Could not automatically find main. ` +
+          `Please manually set your package.json main.`);
+    }
+  }
+
+  if (!packageJson.author &&
+      (!packageJson.contributors || packageJson.contributors.length === 0)) {
+    const npmAuthors = [];
+    // Some Polymer elements use `author` even though the bower.json spec only
+    // specifies `authors`. Check both.
+    const bowerAuthors = bowerJson.authors || (bowerJson as any).author || [];
+    for (const bowerAuthor of bowerAuthors) {
+      if (typeof bowerAuthor === 'string') {
+        npmAuthors.push(bowerAuthor);
+      } else {
+        npmAuthors.push({
+          name: bowerAuthor.name,
+          email: bowerAuthor.email,
+          url: bowerAuthor.homepage,  // The only difference in the specs.
+        });
+      }
+    }
+    if (npmAuthors.length === 1) {
+      packageJson.author = npmAuthors[0];
+    } else if (npmAuthors.length > 1) {
+      packageJson.contributors = npmAuthors;
+    }
+  }
+
+  if (!packageJson.license) {
+    let license;
+    if (typeof bowerJson.license === 'string') {
+      license = bowerJson.license;
+    } else if (Array.isArray(bowerJson.license)) {
+      if (bowerJson.license.length === 1) {
+        license = bowerJson.license[0];
+      } else {
+        console.warn(
+            `${bowerJson.name}: Found multiple licenses in bower.json, ` +
+            `but package.json must have only one.`);
+      }
+    }
+    if (license) {
+      if (license.includes('polymer.github.io/LICENSE')) {
```

Comment:
Follow https://docs.npmjs.com/files/package.json#license / https://yarnpkg.com/lang/en/docs/package-json/#toc-license and use `license:  "SEE LICENSE AT polymer.github.io/LICENSE"`

Reply:
Our license is identical to the standard 3-Clause BSD at https://opensource.org/licenses/BSD-3-Clause except that we use the word "owner" instead of "holder" in one place, and we format the list with stars instead of numbers. Pretty annoying, I wonder why we made these trivial differences. Should we really link to our version, or just use the standard SPDX ID? I feel like using the standard ID is going to help with automation.

Reply:
(Also in case it wasn't clear, this was the previous behavior, I did not change this behavior in this PR.)

Reply:
I previously had implemented the conversion. I didn't realize you could do `"SEE LICENSE IN" at that time.

Reply:
We discussed offline and decided to keep the BSD-3-Clause ID.